docs/context.html

```diff
@@ -79,7 +79,7 @@ <h2 id="contact">Contact</h2>
 <a class="sourceLine" id="cb2-7" title="7">@(contact.urns[<span class="dv">0</span>]) → tel:+<span class="dv">12065551212</span></a>
 <a class="sourceLine" id="cb2-8" title="8">@contact.urn → tel:+<span class="dv">12065551212</span></a>
 <a class="sourceLine" id="cb2-9" title="9">@(foreach(contact.groups, extract, <span class="st">&quot;name&quot;</span>)) → [Testers, Males]</a>
-<a class="sourceLine" id="cb2-10" title="10">@contact.fields → {activation_token: AACC55, age: <span class="dv">23</span>, gender: Male, join_date: <span class="dv">2017-12</span>-02T00:<span class="dv">00</span>:<span class="fl">00.000000</span><span class="dv">-02</span>:<span class="dv">00</span>, not_set: }</a>
+<a class="sourceLine" id="cb2-10" title="10">@contact.fields → Activation Token: AACC55\nAge: <span class="dv">23</span>\nGender: Male\nJoin Date: <span class="dv">2017-12</span>-02T00:<span class="dv">00</span>:<span class="fl">00.000000</span><span class="dv">-02</span>:<span class="dv">00</span></a>
```

Comment:
Huh, so this is going to show all unset fields too ya? Wonder since we have full control of that whether we should exclude those.

Reply:
Oh hm, that is weird stying.